### PR TITLE
update to golang 1.13.4

### PR DESCRIPTION
**Pull request by katiewasnothere**
Signed-off-by: Kathryn Baldauf <kabaldau@microsoft.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ os:
 - linux
 
 go:
-  - "1.12.x"
+  - "1.13.4"
```
Comment (kevpar) [inline .travis.yml]: We should keep the same versioning scheme used here probably. So "1.13.x"

File: contrib/Dockerfile.test [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # 3.) $ make binaries install test
 #
 
-ARG GOLANG_VERSION=1.12
+ARG GOLANG_VERSION=1.13.4
```
Comment (kevpar) [inline contrib/Dockerfile.test]: Same here, let's just do "1.13"

File: .travis.yml (hunk 1 above)
Comment (katiewasnothere) [reply .travis.yml]: Upstream is changing this to be explicit, https://github.com/containerd/containerd/pull/3806

File: contrib/Dockerfile.test (hunk 2 above)
Comment (kevpar) [reply contrib/Dockerfile.test]: Actually not sure about this one. It might be referring to the release version 1.13

File: .travis.yml (hunk 1 above)
Comment (kevpar) [reply .travis.yml]: Okay, that's fine then.

File: contrib/Dockerfile.test (hunk 2 above)
Comment (kevpar) [reply contrib/Dockerfile.test]: This looks fine too

**Pull request merged**